ansible/bootstrap.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Bootstrap node servers
+  hosts: all
+  remote_user: master
+  become: yes
+
+  vars:
+    bootstrap_version: master
+    vault_role: ae-node
+    package: "https://s3.eu-central-1.amazonaws.com/aeternity-node-builds/aeternity-latest-ubuntu-x86_64.tar.gz"
+
+  tasks:
+    - name: Clone infrastructure repo
+      git:
+        repo:  https://github.com/aeternity/infrastructure.git
+        dest: /infrastructure
+        version: "{{ bootstrap_version }}"
+        depth: 1
+        force: yes
+
+    - name: Run bootstrap.sh
+      shell: |
+        /infrastructure/scripts/bootstrap.sh \
+         --vault_addr={{ vault_addr }} \
+         --vault_role={{ vault_role }} \
+         --env={{ env }} \
+         --epoch_package={{ package }} \
+         >> /tmp/user-data.log 2>&1
```

Comment:
I don't think it's good to append to that file. I'm not sure if there a better way to handle the output at all.

Reply:
Then maybe append to /tmp/provision.log ? I think it's good to have the output somewhere, because when running the playbook you see nothing, just the status. Another thing i will try is to register output and display it during the play on fail